apps/api-gateway/src/user/user.controller.ts

```diff
@@ -1,9 +1,18 @@
-import { Controller } from '@nestjs/common';
+import { CreateUserDto } from '@apps/user/src/user/dto/create-user.dto';
+import { Body, Controller, Post } from '@nestjs/common';
+import { ApiCreatedResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
 import { UserService } from './user.service';
-
-// swagger 적는 곳
-
 @Controller('user')
+@ApiTags('User API')
 export class UserController {
-  public constructor(private readonly usersService: UserService) {}
+  public constructor(private readonly userService: UserService) {}
+
+  @ApiOperation({ summary: '신규 사용자 추가' })
+  @ApiCreatedResponse({
+    description: '신규 사용자 추가',
+  })
+  @Post()
+  public async createUser(@Body() createUserDto: CreateUserDto) {
+    return await this.userService.createUser(createUserDto);
```

Comment:
```suggestion
  @ApiOperation({ summary: '신규 사용자 추가' })
  @ApiCreatedResponse({
    description: '신규 사용자 추가',
    type: CreateUserResDto
  })
  @Post()
  public async createUser(@Body() createUserDto: CreateUserDto) {
    return await this.userService.createUser(createUserDto);
```

Swagger를 위한 코드가 있으면 좋을 것 같아요!
지금은 Create에 대해 description 만 존재하는데,
example 데이터나, dto 를 활용한 Type Data 가 Swagger 에서 보여지게 하는 부분이 필요한 거 같습니다!

Reply:
넵!! 피드백 주신대로 controller에 swagger를 위한 operations 코드 추가하였습니다! https://github.com/GrebbMe/grabbme-be/pull/42/commits/3ad44149169577ed30c1763653e576ac7de215a9

해당 내용은 PR에도 추가해놓겠습니다 !!